spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+feature 'User registration' do
+
+  scenario 'User can register as a new user' do
+    user = build :user
+    expect { sign_up(user) }.to change(User, :count).by(1)
+    expect(page).to have_content "Welcome, James Bond"
+    expect(User.first.email).to eq"james.bond@mi6.com"
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.